cpp/splinepy/splines/splinepy_base.cpp

```diff
@@ -439,6 +439,12 @@ void SplinepyBase::SplinepyDerivative(const double* para_coord,
                                       SplinepyWhatAmI());
 };
 
+void SplinepyBase::SplinepyJacobian(const double* para_coord,
+                                    double* jacobians) const {
+  splinepy::utils::PrintAndThrowError("SplinepyEvaluate not implemented for",
```

Comment:
```suggestion
  splinepy::utils::PrintAndThrowError("SplinepyJacobian not implemented for",
```
copy and paste side effects

Reply:
You have seen nothing
...